packages/core/redux/tokens/actions.js

```diff
@@ -323,3 +316,37 @@ export const estimateBtcFee = (params, callback) => async () => {
   }
 }
 
+export const getWalletBalances = ({ wallet }) => {
```

Comment:
This is not an action. Please move this method to 'utils.js' or another appropriate file